README.md

```diff
@@ -1,9 +1,15 @@
 # team-snitch-hackathon
 A Hogwarts House Cup App for Slack
 
+# Run the entire application end-to-end:
+- `docker-compose up -d`
 
 # Running the backend:
 - `cd api && pip3 install pipenv`
 - `pipenv install`
 - `pipenv shell`
 - `python3 app.py`
+
+# Running the backend with Docker:
+- `cd api && docker build -t api .`
```

Comment:
Should maybe mention versioning rather than overwriting the image every time, but then again this is hack-a-thon.

Reply:
yeah this is just for dev. the mounted volumes should update the application in the container when there's code changes

Reply:
Gotcha coolio :)